tests/apidoc_room_state_test.go

```diff
@@ -0,0 +1,111 @@
+package tests
+
+import (
+	"crypto/rand"
+	"io/ioutil"
+	"math/big"
+	"net/url"
+	"testing"
+
+	"github.com/tidwall/gjson"
+
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/client"
+	"github.com/matrix-org/complement/internal/match"
+	"github.com/matrix-org/complement/internal/must"
+)
+
+func TestRoomState(t *testing.T) {
+	deployment := Deploy(t, b.BlueprintAlice)
+	defer deployment.Destroy(t)
+	authedClient := deployment.Client(t, "hs1", "@alice:hs1")
+	t.Run("Parallel", func(t *testing.T) {
+		// sytest: GET /rooms/:room_id/state/m.room.member/:user_id fetches my membership
+		t.Run("GET /rooms/:room_id/state/m.room.member/:user_id fetches my membership", func(t *testing.T) {
+			t.Parallel()
+			roomID, _ := roomFixture(t, authedClient)
```

Comment:
I don't understand why we need `roomFixture` when `roomID := authedClient.CreateRoom(t, map[string]interface{}{})` is not only clearer but doesn't need `_` on the alias.